dev-docs/plugins/bc/bc-prebid-plugin-prebid-options.md

```diff
@@ -519,6 +519,72 @@ http://acdn.adnxs.com/video/plugins/bc/prebid/bc_prebid_vast_plugin.min.js
 
 `options1.prebidPluginPath = 'https://your-path/bc_prebid_vast_plugin.js';`
 
+<a name="pb-adapters"></a>
+### adapters
+
+**Description:**
+
+Adapters are a mechanism that a publisher can use to add some specific behavior at runtime to customize the behavior of the Prebid plugin.  
+
+The `adapters` option defines a list of adapters that the publisher would like the Prebid plugin to load and execute.
+
+See [How To Build An Adapter for Prebid Plugin]({{site.baseurl}}/dev-docs/plugins/bc/bc-prebid-plugin-building-adapter.html) for details on how to build an adapter.
+
+**Acceptable Values:**
+
+If specified, the value of this option should be an array containing one or more adapter definitions.
+
+Each adapter definition is specified in object with the following fields:
```

Comment:
Should probably say "Each adapter definition is specified as an object with the following fields:"

Reply:
fixed